src/requirements/Allowlist/AllowlistRequirement.tsx

```diff
@@ -75,6 +90,23 @@ const AllowlistRequirement = ({ ...rest }: RequirementProps): JSX.Element => {
         </Button>
       )}
       <SearchableVirtualListModal
+        aboveList={
```

Comment:
I was about to say that maybe we could maybe do this in a more composable way, but that would require a `SearchableVirtualListModal` refactor, and afaik this is not the final solution for this (you mentioned some kind of checkmark icon/animation if the address is included in the allowlist), so we can keep it this way for now.

Reply:
Good idea, thanks for sharing! We should brainstorm on this, once we have a clear idea on what to do with the modal!